intel_extension_for_transformers/llm/runtime/graph/vectors/cpu/vec_load.cpp

```diff
@@ -13,38 +13,3 @@
 //  limitations under the License.
 
 #include "vec_load.hpp"
```

Comment:
this vec_load.cpp become empty file for now, do we still need keep this cpp file ?

Reply:
Yes, we should compile a static library by `.cpp`